Assets/InputSimulationHelpGuide.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    public class InputSimulationHelpGuide : MonoBehaviour
+    {
+        [SerializeField]
+        [Tooltip("Additional distance on top of sphere cast radius when pointer is considered 'near' an object and far interaction will turn off")]
+        private GameObject ShortcutTip = null;
```

Comment:
Per our guidelines, it would be nice to make these configurable via code if they are going into core MRTK.